lib/rules/at-rule-empty-line-before/index.js

```diff
@@ -59,7 +59,7 @@ const rule = function(expectation, options, context) {
       const isNested = atRule.parent !== root;
 
       // Ignore the first node
-      if (atRule === root.first) {
+      if (atRule === root.first || atRule === atRule.root().first) {
```

Comment:
Please add comment around `atRule === atRule.root().first` and good work!

Reply:
Done.